e2e/ansible/playbooks/resiliency/test-disk-failure/test-disk-failure.yml

```diff
@@ -0,0 +1,221 @@
+           regex1: "{{replace_item}}"
+           regex2: "{{replace_with}}"
+
+       - name: 5) Create the Namespace
+         shell: kubectl create ns {{ namespace }}
+         args:
+           executable: /bin/bash
+         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+
+       - name: 6) Create the storage pool path
+         shell: source ~/.profile; kubectl apply -f storage-pool-path_creation.yml  -n {{ namespace }}
+         args:
+           executable: /bin/bash
+         register: result_sp_path
+         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+
+       - name: 7) Obtaining storage classes yaml
+         shell: kubectl get sc openebs-percona -o yaml > "{{ create_sc }}"
+         args:
+           executable: /bin/bash
+         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+
+       - name: 7a) Replace storage class file with test parameters
+         replace:
+           path: "{{ result_kube_home.stdout }}/{{ create_sc }}"
+           regexp: 'openebs.io/storage-pool: default'
+           replace: 'openebs.io/storage-pool: disk'
+         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+
+       - name:  7b) Delete the existing storage class and create new one
+         shell: kubectl delete sc openebs-percona; kubectl apply -f "{{ create_sc }}"
+         args:
+           executable: /bin/bash
+         register: result_sc_out
+         until: "'created' in result_sc_out.stdout"
+         delay: 10
+         retries: 5
+         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+
+       - name: 8) Obtain the sql test script file
+         get_url:
+           url: "{{ sql_test_link }}"
+           dest: "{{ result_kube_home.stdout }}"
+         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+
+       - name: 8a) Create the configmap test
+         command: kubectl create configmap sqltest --from-file=sql-test.sh -n {{ namespace }}
+         args:
+           executable: /bin/bash
+         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+
+       - name: 9) Obtaining the percona liveliness application yaml
+         get_url:
+           url: "{{ percona_link }}"
+           dest: "{{ result_kube_home.stdout }}"
+         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+
+       - name: 9a) Create percona deployment with OpenEBS storage
+         include_tasks: "{{ansible_env.HOME}}/{{utils_path}}/deploy_task.yml"
+         vars:
+           app_yml: "{{ percona_files.0 }}"
+           ns: {{ namespace }}
+
+       - name: 9b) Check whether percona application is running
+         include_tasks: "{{ansible_env.HOME}}/{{utils_path}}/deploy_check.yml"
+         vars:
+           app: percona
+           ns: {{ namespace }}
+
+       - name: 9c) Check Replica pod status
+         include_tasks: "{{ansible_env.HOME}}/{{utils_path}}/deploy_check.yml"
+         vars:
+           app: pvc
+           ns: {{ namespace }}
```

Comment:
Do we have a check to confirm that the pods are running on the specified storage pool path ?